py/DREAM/Output/LCFSLoss.py

```diff
@@ -54,4 +54,4 @@ def plotScrapedOffJre(self, r=None, t=None, **kwargs):
         
     def plotScrapedOffIre(self, r=None, t=None, **kwargs):
```

Comment:
The argument ``r`` is not used anymore, so it could be removed from this function's arguments

Reply:
Good catch! This was a bug since before. Since ``Ire`` is a scalar (in space), giving a radial coordinate makes no sense :P